spring-test/src/main/java/org/springframework/test/context/event/package-info.java

```diff
@@ -0,0 +1,7 @@
+/**
+ * Test event support classes for the <em>Spring TestContext Framework</em>.
+ *
+ * @author Frank Scheffler
+ * @since 5.2
```

Comment:
Please remove the `@author` and `@since` tags since we do not use those for packages in Spring.

Reply:
Please add `package-info.java` for the new `annotation` package.